scalafmt-core/shared/src/main/scala/org/scalafmt/config/ContinuationIndent.scala

```diff
@@ -15,6 +15,7 @@ case class ContinuationIndent(
     defnSite: Int = 4,
     caseSite: Int = 4,
     ctorSite: Option[Int] = None,
+    @annotation.ExtraName("deriveSite")
```

Comment:
by the way, do we need to mention this in the documentation?

Reply:
We don't, I was thinking to go through all the PRs and update it all just before the release. Does it make sense or should I update it now?